tools/mkrepo

```diff
@@ -81,7 +82,7 @@ function mkdir(d) {
 
 function gitify(dir, repo) {
     exec('git init', dir, function () {
-        exec('git remote add origin git@github.com:joyent/' + repo + '.git',
+        exec('git remote add origin https://github.com/TritonDataCenter/' + repo + '.git',
```

Comment:
I think we should probably leave this as ssh here (and probably elsewhere in this PR?). I don't have my gh token set up for doing operations over HTTP, but my ssh key is always available.